pkg/devcontainer/docker_client.go

```diff
@@ -0,0 +1,36 @@
+package devcontainer
+
+import (
+	"context"
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
```

Comment:
hm, seems like it's now `github.com/moby/moby`

Reply:
It's complicated :)